tasks/managed/embargo-check/embargo-check.yaml

```
@@ -111,7 +111,7 @@ spec:
             > "$(workspaces.data.path)"/ir-result.txt || \
             (grep "^\[" "$(workspaces.data.path)"/ir-result.txt | jq . && exit 1)
 
-        internalRequest=$(awk 'NR==1{ print $2 }' "$(workspaces.data.path)"/ir-result.txt | xargs)
+        internalRequest=$(grep created "$(workspaces.data.path)"/ir-result.txt | awk 'NR==1{ print $2 }' | xargs)
```

I never liked using `xargs` (which is a tool for running utilities with multiple arguments) just to strip `''`. So how about this?
```suggestion
        internalRequest=$(awk -F"'" '/created/ { print $2 }' "$(workspaces.data.path)"/ir-result.txt)
```
What this does is it sets `'` as the field delimiter and then prints the second part. Thanks to `/created/` it will only use lines with `"created"`.

Another option is combination of `grep` and `cut`:
```
        internalRequest=$(grep created "$(workspaces.data.path)"/ir-result.txt | cut -d"'" -f 2)
```

---

The first one works just fine for me locally, so I will switch to that

---

updated

---

cool, thanks